docs/source/en/training/distributed_inference.md

```diff
@@ -52,6 +52,62 @@ To learn more, take a look at the [Distributed Inference with 🤗 Accelerate](h
 
 </Tip>
 
+### Using `device_map`
+
+<Tip warning={true}>
+
+🧪 This feature is experimental and its APIs might change in future. 
+
+</Tip>
+
+With 🤗 Accelerate, you can make use of a supported `device_map` strategy to distribute the model-level components of a pipeline across multiple devices. The section below explains a situation where performing distributed inference via `device_map` could be useful.
+
+Let's say you have two consumer GPUs each having 8GBs of VRAM. Using `enable_model_cpu_offload()` might not work best here because -
+
+* it works only on a single GPU
+* a single model might not fit on a single GPU `enable_sequential_cpu_offload()` might work but will be extremely slow. Here also we're limited to using a single GPU only.
```

Comment:
```suggestion
For example, if you have two 8GB GPUs, then using [`~DiffusionPipeline.enable_model_cpu_offload`] may not work so well because:

* it only works on a single GPU
* a single model might not fit on a single GPU ([`~DiffusionPipeline.enable_sequential_cpu_offload`] might work but it will be extremely slow and it is also limited to a single GPU)
```

Reply:
If I understand this PR correctly, each model is completely assigned to a GPU, or to the CPU as a whole. How does this solve the problem of a single model not fitting on a single GPU?

Reply:
If it doesn't fit on a single GPU it's offloaded to the CPU, yes. If it doesn't fit on the CPU, yes that is going to be a problem for now. We will error out in that case (have noted that in the docs). This is gonna be improved in the future iterations too (disk offloading) if there's enough demand. 

If you're thinking about model sharding across GPU, CPU, etc. then it's a separate PR: https://github.com/huggingface/diffusers/pull/6396